decidim-core/db/seeds.rb

```diff
@@ -54,6 +54,7 @@
       port: ENV.fetch("SMTP_PORT", nil) || ENV.fetch("DECIDIM_SMTP_PORT", "25")
     },
     host: ENV.fetch("DECIDIM_HOST", "localhost"),
+    secondary_hosts: ENV.fetch("DECIDIM_HOST", "localhost") == "localhost" ? ["0.0.0.0", "127.0.0.1"] : nil,
```

Comment:
```suggestion
    secondary_hosts: ["0.0.0.0", "127.0.0.1"],
```

Reply:
I would prefer to respect the check to the DECIDIM_HOST env var, as if not this would add some noise to the seeded database in cases where the env var is set-up, and it may make new implementers to doubt what's this setting about (in system panel). 

What I mean is that someone can set up a staging server (for instance, `try.decidim.org`) and then it'd have the secondary_hosts always set-up. 

The case of  external_domain_whitelist is different because it serves as a demonstration on what this feature can do.

In this line, I guess that we could add the `www` prefix just in case someone wants to add it (i.e. `www.try.decidim.org`). I'm like 95% sure this was the original reason of this feature (`www.decidim.barcelona`). 

```ruby
ENV.fetch("DECIDIM_HOST", "localhost") == "localhost" ? ["0.0.0.0", "127.0.0.1"] : "www.#{ENV.fetch("DECIDIM_HOST")}"
```

Maybe that's enough?

Reply:
No, based on your arguments, i would go with the initial implementation.